### DISCUSSION TITLE
Am I on the wrong OFW?

### DISCUSSION THREAD

**Discussion by tzukiwy** [Th3Un1q3/flipp_pomodoro#205] [General]
I'm on the "Release" branch version 0.84.2. When I copied the .fap into /apps/Misc/ and run it through Apps, I get the API mismatched error. Am I doing something wrong?

**Comment by Th3Un1q3** [comment]
@tzukiwy, I've responded in the [issues](https://github.com/Th3Un1q3/flipp_pomodoro/issues/207) section. Please continue the discussion over there.
